assets/css/_events.scss

```diff
@@ -181,10 +181,6 @@ $mobile-control-height: 73px;
   cursor: pointer;
 }
 
-.hidden {
-  display: none !important;
```

Comment:
This feels like it could be a big risky change, but also this utility was overriding the tailwind utility that does the same thing (except that the tailwind utility doesn't have `!important` anymore, since we removed that [here](https://github.com/mbta/dotcom/commit/3c6456cac09cd3359d0287de1bc3eeed07a26329#diff-52a43e64271620d979d25d597733b7930e6f3771ab30e71ec459bff2398e43dbR19)).

Interested in your thoughts on getting rid of this, and whether there's another way to get tailwind to conditionally give us `display: none` in the right settings.